testing/e2e/app/routes/Panels/spec.ts

```diff
@@ -0,0 +1,254 @@
+import { test, expect, Locator } from '@playwright/test';
+
+test('should display initialActiveId as the initial panel', async ({
+  page,
+}) => {
+  await page.goto('/Panels');
+
+  await expect(page.locator('#root')).toBeVisible();
+  await expect(page.locator('#more-info')).not.toBeVisible();
+});
+
+test('should animate to the next panel when the trigger is clicked', async ({
```

Comment:
You're right, it makes more sense to highlight the playwright test for my comment above. I was just thinking if it was necessary to have a test where you switch back and forth between panels to make sure the `Back` button works properly.